roles/mesos/defaults/main.yml

```diff
@@ -8,7 +8,8 @@ mesos_agent_package: "mesos-agent-{{ mesos_version }}"
 mesos_mode: follower
 mesos_log_dir: /var/log/mesos
 mesos_logging_level: WARNING
-mesos_work_dir: /var/lib/mesos
+mesos_work_dir: /mnt/mesos
+mesos_lvm_data_volume_size: 20%FREE
```

Comment:
It can probably will vary a lot based on load and the specific apps, but here is some data from a cluster that I have access to that has been running for about a month:

```
391M    /var/lib/mesos/
3.7G    /var/lib/mesos/
1.5G    /var/lib/mesos/
694M    /var/lib/mesos/
1.1G    /var/lib/mesos/
188K    /var/lib/mesos/
188K    /var/lib/mesos/
184K    /var/lib/mesos/
2.7G    /var/lib/mesos/
430M    /var/lib/mesos/
422M    /var/lib/mesos/
473M    /var/lib/mesos/
3.0G    /var/lib/mesos/
1.9G    /var/lib/mesos/

117M    /var/lib/docker/
116M    /var/lib/docker/
118M    /var/lib/docker/
120M    /var/lib/docker/
45M     /var/lib/docker/
45M     /var/lib/docker/
1.1G    /var/lib/docker/
1.8G    /var/lib/docker/
3.4G    /var/lib/docker/
7.0G    /var/lib/docker/
2.3G    /var/lib/docker/
1.1G    /var/lib/docker/
11G     /var/lib/docker/
3.3G    /var/lib/docker/
3.9G    /var/lib/docker/
3.8G    /var/lib/docker/
2.1G    /var/lib/docker/
9.4G    /var/lib/docker/
10G     /var/lib/docker/
12G     /var/lib/docker/
9.3G    /var/lib/docker/
13G     /var/lib/docker/
21G     /var/lib/docker/
```